include/config/pokemon.h

```diff
@@ -37,10 +37,11 @@
 #define P_SHEDINJA_BALL             GEN_LATEST  // Since Gen 4, Shedinja requires a Poké Ball for its evolution. In Gen 3, Shedinja inherits Nincada's Ball.
 #define P_KADABRA_EVERSTONE         GEN_LATEST  // Since Gen 4, Kadabra can evolve even when holding an Everstone.
 #define P_SHUCKLE_BERRY_JUICE       GEN_LATEST  // In Gen 2, Shuckle had a 1/16 chance of converting Berry that it's holding into Berry Juice. Enabling this will allow Shuckle to do this with an Oran Berry, which is the spiritual succesor of the Berry item.
-#define P_ARCEUS_UNIQUE_FORM_ICONS  GEN_LATEST  // Since Gen 9, Arceus additionally changes its icon to reflect its current form.
 
 // Species graphic settings
-#define P_CUSTOM_GENDER_DIFF_ICONS      TRUE        // If TRUE, will give more Pokémon custom icons for their female forms, i.e. Hippopotas and Hippowdon
+#define P_GENDER_DIFFERENCES            TRUE        // If TRUE, Pokémon will have graphical-only differences when they're female. Female *forms* such as Meowstic are not affected by this.
```

Comment:
This is just for overworlds, right?

Reply:
Yeah Idk about getting rid of a separate P_CUSTOM_GENDER_DIFF_ICONS either because it technically isn't a thing any games do, so some hackers might want to disable it separately

Reply:
> This is just for overworlds, right?

No, it's all gender differences, which includes overworlds.

> Yeah Idk about getting rid of a separate P_CUSTOM_GENDER_DIFF_ICONS either because it technically isn't a thing any games do, so some hackers might want to disable it separately

I'm not getting rid of them, they're below.

Reply:
Oh lol, ignore me then